applications/LiisaPortfolioTracker.md

```diff
@@ -0,0 +1,269 @@
+| 1b.    | Wallet address transmission    | Implement a front-end authentication module that captures and transmits wallet addresses from the user sign-in process to our backend. The transmission activates event-driven triggers, initiating Subscan API queries for relevant event data retrieval. This will be developed using Node.js and Bubble.io.                                                                                                             |
+| 2a.    | Dashboard Structure Implementation | Utilizing provided mockups as the architectural blueprint, implement the user interface for the application dashboard. This involves arranging key performance indicators (KPIs) and various data visualization elements according to the predefined design. This will be developed using Bubble.io.                                                                       |
+| 2b.    | Dynamic Dashboard Values       | Implement a real-time data binding mechanism that retrieves and displays updated portfolio data from the database, activated by event-driven triggers as defined in 1b. This will be achieved using a websocket client connected to our PostgreSQL Database.                                                                                      |
+
+## Mockups
+
+### Portfolio Overview
+
+The homepage of the portfolio provides a quick and insightful summary of the user's NFT holdings, including key features such as:
+- Overview Metrics: Displaying estimated portfolio value, revenue from NFT sales, spending on NFT purchases, realized profit-and-loss, and unrealized profit-and-loss;
+- Portfolio Scorecard: Assessing liquidity, volatility, and diversification of the NFT portfolio;
+- Portfolio Evolution Chart: Visualizing the historical value evolution of the NFT portfolio;
+- Inventory Chart: Illustrating the relative value distribution of each NFT holding.
+- Collection Highlights: Ranking collections in the user’s wallet based on top gainers, top losers, and recent purchases.
+
+![Portfolio tracker #1 (1)](https://github.com/LiisaNFT/W3F-Grants-Program/assets/139144686/ae122745-4d96-4c2a-86af-a8bc8230f63a)
+
+### NFTs page
+
+The NFTs page offers users a technical and visually engaging interface to view their NFT collection:
+- NFT Images: Displaying artwork and associated metadata for each token;
+- Token Metrics: Showing estimated value, acquisition price, unrealized PNL, and 24-hour sales for each collection;
+- Filtering and Sorting: Allowing users to sort and filter NFTs based on metrics like collection sales and estimated value.
+
+![Portfolio tracker #1 (1)](https://github.com/LiisaNFT/W3F-Grants-Program/assets/139144686/c8b1dde7-d548-4153-92d7-064b4f0f3ada)
```

Comment:
I think you posted this image already on [L209](https://github.com/w3f/Grants-Program/pull/1855/files#diff-7b9b655bfcc02275c2dc8f9b9bd14841955ca499c300867de9f29ecdb1437b04R209)